### PR TITLE
Add required and optional parameters to QueueAdd

### PR DESCRIPTION
**Pull request by arenddeboer**
Added required and optional parameters to QueueAdd

### CODE REVIEW

File: src/message/action.js [hunk 1]
```diff
@@ -917,8 +919,22 @@ function Originate() {
  * @property {String} StateInterface Optional, State interface
  * @augments Action
  */
-function QueueAdd() {
+function QueueAdd(queue, interface, memberName, penalty, stateInterface) {
 	QueueAdd.super_.call(this, 'QueueAdd');
+    	this.set('queue', queue);
+    	this.set('interface', interface);
```
Comment (marcelog) [inline src/message/action.js]: **interface** is a reserved keyword, let's change the name, I'll be happy to merge :)

File: src/message/action.js [hunk 2]
```diff
@@ -917,8 +919,22 @@ function Originate() {
  * @property {String} StateInterface Optional, State interface
  * @augments Action
  */
-function QueueAdd() {
+function QueueAdd(queue, interface, memberName, penalty, stateInterface) {
```
Comment (marcelog) [inline src/message/action.js]: Let's leave optional arguments out of the function variables, so we can keep the standard nami behavior to add options, like in [this example](https://github.com/marcelog/Nami/blob/master/src/index.js#L73), one only has to instantiate the action and just set whatever fields are needed without having to have a lot of arguments when instantiating the action itself (this allow us to have a cleaner interface)

Comment (arenddeboer) [inline src/message/action.js]: Sounds good, please see my new PR. I took the liberty to rename "interface" in some other places as well.

File: src/message/action.js [hunk 3]
```diff
@@ -879,8 +879,10 @@ function QueueReset() {
  * @property {String} Interface Interface
  * @augments Action
  */
-function QueueRemove() {
+function QueueRemove(queue, asteriskInterface) {
```
Comment (marcelog) [inline src/message/action.js]: Looking great! :) One minor thing: Shouldn't the order of the arguments be the interface first and the queue next? Same for QueueAdd. So it matches QueuePause and QueueUnpause

Comment (arenddeboer) [inline src/message/action.js]: It depends if you want to follow the official Asterisk parameter order. It does seem strange that just for the pause function, Asterisk switches the queue/interface parameter order though. 
Happy to go either way. 
(I do see where you're coming from setting the variables outside the action call :-) )

Comment (marcelog) [inline src/message/action.js]: Oh I see your point. I was actually refering to keeping consistency inside nami itself by using the same arguments in the same order. Makes sense?

Comment (arenddeboer) [inline src/message/action.js]: It sure does

**Comment by marcelog** [comment]
Thanks! :)

**Comment by arenddeboer** [comment]
Thank you!  great lib

**Pull request merged**